src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -638,9 +633,54 @@ public void addRtpExtension(RtpExtension rtpExtension)
     }
 
     @Override
-    public void setVideoConstraints(Map<String, VideoConstraints> newVideoConstraints)
+    public void setSenderVideoConstraints(ImmutableMap<String, VideoConstraints> newVideoConstraints)
     {
+        ImmutableMap<String, VideoConstraints>
+            oldVideoConstraints = bitrateController.getVideoConstraints();
+
         bitrateController.setVideoConstraints(newVideoConstraints);
+
+        Set<String> endpointsThatNoLongerCare
+            = new HashSet<>(oldVideoConstraints.keySet());
+        endpointsThatNoLongerCare.removeAll(newVideoConstraints.keySet());
+
+        // Endpoints that no longer care what they receive.
```

Comment:
The method sets the constraints that `this` wishes to impose on other senders. So this must be the set of ID that `this` "doesn't care" about.

Reply:
You're right, the comment in the code is wrong. How about this `// Endpoints that "this" no longer cares about what it receives.`.

And also rename the variable to `endpointsINoLongerCare`?

Reply:
Makes sense. `removedEndpoints`?